msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/DefaultRetryPolicy.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+/**
+ * Default retry policy for most MSAL Java flows
+ */
+class DefaultRetryPolicy implements IRetryPolicy {
+    private static final int RETRY_NUM = 1;
+    private static int RETRY_DELAY_MS = 1000;
+
+    @Override
+    public boolean isRetryable(IHttpResponse httpResponse) {
+        return httpResponse.statusCode() >= 500 &&
+                httpResponse.statusCode() < 600 &&
```

Comment:
There should already be constants defined for these numbers? See in msal-node: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/src/utils/Constants.ts#L62

Reply:
In the latest commit there's a new `HttpStauts` enum with a variety of HTTP status codes. To keep this PR simple I only used them to the RetryPolicy classes, but they will be used in other classes as they get expanded/refactored.